src/containers/participant/AddNewPlanStatusModal.js

```diff
@@ -134,7 +134,7 @@ const AddNewPlanStatusModal = ({ isVisible, onClose } :Props) => {
       updatedFormData = setIn(updatedFormData, effectiveDateKeyPath, effectiveDateAsDateTime);
     }
 
-    const associations = [
+    const associations :Array<Array<*>> = [
```

Comment:
No change required, just a suggestion. It's not hard to avoid the `*` so if you really want to type this, you should declare in `types.js` and import it where it's used. It would look something like this:

```
type Associations = Array<[string, UUID | number, string, UUID | number, string, Object]>
```

It's probably not 100% accurate, but you get the idea.

Reply:
oh yaaa this makes sense. i'll do this sort of thing going forward!